routes/types.py

```diff
@@ -65,6 +66,21 @@ def rd_metadata(self):
     def rd_type(self):
         return self.rd_metadata.get("type", ResultType.ROUTE.value)
 
+    # Try to extract a route from the result, it's not in the metadata and has to be parsed from the line
+    @property
+    def rd_route(self):
+        if "$PATH" in self.metavars: #For some rules we have extracted a path in the $PATH metavar
+            return "{} {}".format(self.rd_method, self.metavars['$PATH']['abstract_content'])
+        elif self.rd_type == ResultType.ROUTE.value:
+            return self.first_line
+        else:
+            return None
+    
+    @property
+    def rd_method(self):
+        # split by space and deduplicate and return first element
+        return list(set(self.metavars['$METHOD']['abstract_content'].split(' ')))[0]
```

Comment:
```suggestion
        return list(set(self.metavar_content('$METHOD').split(' ')))[0]
```

Reply:
Why split on `' '` here and take the first element? It seems cleaner to me to just use the `$METHOD` content.